app-crypt/pinentry/pinentry-1.1.1-r1.ebuild

```diff
@@ -52,7 +52,7 @@ src_prepare() {
 }
 
 src_configure() {
-	[[ "$(gcc-major-version)" -ge 5 ]] && append-cxxflags -std=gnu++11
+	append-cxxflags -std=gnu++11
```

Comment:
GCC 11.x defaults to C++17, so this is downgrading the C++ standard used (i.e. this could probably be dropped entirely now, as older GCC already defaults to C++11 standard in any case).

Reply:
Yep -- let's do it.

Reply:
Done, but worried there might be quite a few more of these. I already found some in the past that were intended to help with Qt compatibility but now end up setting us backwards :(